### ISSUE TITLE
Есть ли план поддержки  macOS 10.14 

### ISSUE THREAD

**Issue by nipisarev** [packagesdev/savehollywood#3]
Now I get it "You cannot use the SaveHollywood screen saver with this version of OS X."

**Comment by dncrews** [comment]
I discovered today that if it's already set up before you upgrade to Mojave, it still works with the same configuration you had before. It shows that message on the configuration page, though, so you can't change the configuration at all.

**Comment by HeminWon** [comment]
@nipisarev You are right, I think it`s macOS 10.14 BUG. because all Screen Saver are not working

**Comment by packagesdev** [comment]
Screen Saver support is a bit bette in the latest beta but there are still some bugs in macOS Mojave so far. I will add support for Dark Appearance in the Settings UI in a future minor update.

**Comment by packagesdev** [comment]
http://s.sudre.free.fr/Software/SaveHollywood/release_notes.html

But there are still really annoying bugs in macOS Mojave when it comes to the Desktop & Screen Saver pref pane. It's as if they haven't tested it at all.
